
bool STATUS_PRE_RESUME
bool STOP_RESUME
bool STATUS_RUNNING
